cartographer/io/submap_painter.h

```diff
@@ -55,10 +58,43 @@ struct SubmapSlice {
   int metadata_version = -1;
 };
 
+struct SubmapTexture {
+  struct Pixels {
+    std::vector<char> intensity;
+    std::vector<char> alpha;
+  };
+  Pixels pixels;
```

Comment:
Is there some special reason, why this is not "std::vector<Pixel> pixels;", where
struct Pixel {
  char intensity;
  char alpha;
};
?